spec/models/user_spec.rb

```diff
@@ -10,4 +10,62 @@
 
     expect(user.interests.count).to eq 2
   end
+
+  describe '#to_csv' do
+    it 'returns user interests list' do
+      create(:user, email: 'example@mail.com', is_male: true)
+
+      expect(User.to_csv).to eq "id,email,gender,age,interests_list\n1,example@mail.com,male,23,\"\"\n"
+    end
+  end
+
+  describe '#age' do
+    context 'when user did not have a birthday this year' do
+      it 'it returns the current age in years of a given user' do
+        travel_to Date.new(2018, 1, 30) do
+          user = create(:user, birthday: '1994-08-21')
+
+          expect(user.age).to eq 23
+        end
+      end
+    end
+
+    context 'when user did have a birthday this year' do
+      it 'it returns the current age in years of a given user' do
+        travel_to Date.new(2018, 9, 11) do
+          user = create(:user, birthday: '1994-08-21')
+
+          expect(user.age).to eq 24
+        end
+      end
+    end
+  end
+
+  describe '#interests_list' do
+    it 'returns user interests list' do
+      user = create(:user)
+      create(:interest, user: user, name: 'Rails')
+      create(:interest, user: user, name: 'Ruby')
+
+      expect(user.interests_list).to eq 'Rails, Ruby'
+    end
+  end
+
+  describe '#gender' do
+    context 'when user is_male field equal false' do
```

Comment:
Why not just use `when user is female?`? `is_male`field is an implementation detail, and we aim to use specs to describe behaviour, not implementation